docs/source/AdministratorGuide/Resources/proxyprovider.rst

```diff
@@ -0,0 +1,85 @@
+.. _resourcesProxyProvider:
+
+==============
+ProxyProvider
+==============
+
+This resource type provides an interface to obtain proxy certificates using a user identifier. The following proxy providers are presented here: ``DIRACCA``, ``PUSP``.
+
+
+DIRACCA proxy provider
+----------------------
+
+ProxyProvider implementation for the proxy generation using local Certification Authority (CA) credentials. This class is a simple CA, its main purpose is to generate a proxy on the fly for DIRAC users who do not have a certificate registered in DIRAC registry. To use it need to have CA certificate and key locally. Required parameters in the DIRAC configuration for its implementation::
+
+    Resources
+    {
+      ProxyProviders
+      {
+        MY_DIRAC_CA
+        {
+          ProviderType = DIRACCA
+          CertFile = /opt/dirac/pro/etc/grid-security/MY_DIRAC_CA.pem
+          KeyFile = /opt/dirac/pro/etc/grid-security/MY_DIRAC_CA.key.pem
+          Match = O, OU
+          Supplied = CN
+          Optional = emailAddress
+          DNOrder = O, OU, CN, emailAddress
+        }
+      }
+    }
+
+Configuration options are:
+
+* ``ProviderType``: main option, to show which proxy provider type you want to register. In the case of the DIRACCA proxy provider you must type ``DIRACCA``.
+* ``CertFile``: the path to the CA certificate.
+* ``KeyFile``: the path to the CA key.
+* ``CAConfigFile``: the path to the openssl configuration file. This is optional and not recomended to use. But if you choose to use this option, it is recommended to use a relatively simple configuration.
+* ``Match``: the distinguished name fields that must contain the exact same contents as that field in the CA's DN.
+* ``Supplied``: the distinguished name fields list that must be present.
+* ``Optional``: the distinguished name fields list that are allowed, but not required.
+* ``DNOrder``: order of the distinguished name fields in a created user certificate.
+* ``<One of the supported distinguished names>``: to set default value for distinguished name field, for example.: ``OU = MY, DIRAC``.
+
+Also, as an additional feature, this class can read properties from a simple openssl CA configuration file. To do this, just set the path to an existing openssl configuration file as a CAConfigFile parameter. In this case, the distinguished names order in the created proxy will be the same as in the configuration file policy block.
+
+The Proxy provider supports the following distinguished names (more details about distinguished names here: https://www.cryptosys.net/pki/manpki/pki_distnames.html):
+* SN(surname)
```

Comment:
Need an empty line before the bullet list
```suggestion

* SN(surname)
```

Reply:
done